src/sentry/static/sentry/app/views/organizationRepositories.jsx

```diff
@@ -78,6 +81,14 @@ class AddRepositoryLink extends PluginComponentBase {
     );
   }
 
+  parseGitHubRepo(repo) {
+    let re = /github.com\/(.+\/.+)/i;
```

Comment:
This will also capture values such as `github.com/foo/bar/baz` or `foogithub.com/foo/bar/baz`. So unsure exactly if this will cause a problem or not, but if we're going to be writing tests, we should assert that it also doesn't capture false positives.